components/post_markdown/system_message_helpers.tsx

```diff
@@ -194,8 +198,8 @@ function renderHeaderChangeMessage(post) {
                         username,
                         old: oldHeader,
                         new: newHeader,
-                        strong: (chunks) => (<strong>{chunks}</strong>),
-                        br: (x) => (<><br/>{x}</>),
+                        strong: (chunks: Record<string, any>) => (<strong>{chunks}</strong>),
+                        br: (x: Record<string, any>) => (<><br/>{x}</>),
```

Comment:
The types for both of these can be `React.ReactNode` instead of using a `Record`